conanfile.py

```diff
@@ -26,7 +26,7 @@ class ResultConan(ConanFile):
     generators = "cmake"
 
     # Dependencies
-    build_requires = ("Catch2/2.7.1@catchorg/stable")
```

Comment:
Let me know if you don't like this to be in the same MR. I just had trouble downloading the older version using Conan for some reason, so I switched to the most recent v2 available at conan.io ... Not really related to the rest, though

Reply:
Ordinarily I'd prefer a change like this to be independent, but I think you made the right call. 

I think the version discrepancy comes from Bintray having been sunset since this repository released, and I haven't updated any infrastructure since.